lib/graphql/language/lexer.rl

```diff
@@ -100,6 +104,9 @@
 
     BLANK   => { meta[:col] += te - ts };
 
+
+    UNMATCHED_BLOCK_STRING => { emit(:UNMATCHED_BLOCK_STRING, ts, te, meta) };
+    UNMATCHED_QUOTED_STRING => { emit(:UNMATCHED_QUOTED_STRING, ts, te, meta) };
```

Comment:
What happens when these tokens are passed to the parser? I guess we get some kind of error, but is it a helpful one? Could we add a test to describe this case?